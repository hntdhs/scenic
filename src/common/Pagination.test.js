import React from "react";
import { render } from "@testing-library/react";
import Pagination from "./Pagination";

it("matches snapshot", function () {
    const { asFragment } = render(<Pagination nPages={3}
      currentPage={1}
      setCurrentPage={() => {}}/>);
    expect(asFragment()).toMatchSnapshot();
  });
