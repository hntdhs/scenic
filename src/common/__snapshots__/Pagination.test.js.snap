// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav>
    <ul
      class="pagination pagination-lg"
    >
      <li
        class="page-item"
      >
        <a
          class="page-link"
          href="#"
        >
          Previous
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          class="page-link"
          href="#"
        >
          1
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          class="page-link"
          href="#"
        >
          2
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          class="page-link"
          href="#"
        >
          3
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          href="#"
        >
          Next
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
